.template/spec/addons/base/github/template_spec.rb

```diff
@@ -5,7 +5,65 @@
 
   it 'creates Github actions workflows' do
     expect(file('.github/workflows/deploy_heroku.yml')).to exist
+    expect(file('.github/workflows/publish_wiki.yml')).to exist
+    expect(file('.github/workflows/review_code.yml')).to exist
     expect(file('.github/workflows/test_production_build.yml')).to exist
     expect(file('.github/workflows/test.yml')).to exist
   end
+
+  it 'creates Github wiki structure' do
+    expect(file('.github/wiki/_Sidebar.md')).to exist
+    expect(file('.github/wiki/_Footer.md')).to exist
+    expect(file('.github/wiki/Home.md')).to exist
+    expect(file('.github/wiki/assets/images/.keep')).to exist
+  end
+
+  xit 'modifies the README.md' do
```

Comment:
Can't test this as we're testing against the docker container and this file is in `.dockerignore`, So there is no `README.md` in the container.

Reply:
Added a TODO.